charts/canvas-oda/values.yaml

```diff
@@ -364,7 +364,7 @@ pdb-management-operator:
   enabled: true
   image: tmforumodacanvas/pdb-management-operator
   version: 1.0.0
-  prereleaseSuffix: -alpha
+  prereleaseSuffix:
```

Comment:
Please do not remove the prereleaseSuffix until the BDD tests where executed successfully.
For details see https://github.com/tmforum-oda/oda-canvas/blob/main/docs/developer/work-with-dockerimages.md